### PR TITLE
new datepicke reverted code back for stage

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

https://app.clickup.com/t/1nefg9

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by prateek-giddh — APPROVED** [review]

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 1]
```diff
@@ -30,15 +30,14 @@
                             </ng-template>
                         </sh-select>
                     </div>
-                    <div class="input-group">
-                        <span class="input-wrapper">
-                            <input type="text" name="dateRange" daterangepicker [options]="datePickerOptions"
-                                (applyDaterangepicker)="onDateChange($event)" class="form-control cursor-pointer" />
-                        </span>
+                    <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
+                        <i class="icon-calender calendar-addon"></i>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:34]: sometimes svg is added and sometimes icon, please use 1 thing only at all places

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/manufacturing/report/mf.report.component.ts [hunk 2]
```diff
@@ -267,4 +296,56 @@ export class MfReportComponent implements OnInit, OnDestroy {
...
+            this.selectedRangeLabel = value.name;
+        }
+        this.hideGiddhDatepicker();
+        if (value && value.startDate && value.endDate) {
+            this.selectedDateRange = { startDate: moment(value.startDate), endDate: moment(value.endDate) };
+            this.selectedDateRangeUi = moment(value.startDate).format(GIDDH_NEW_DATE_FORMAT_UI) + " - " + moment(value.endDate).format(GIDDH_NEW_DATE_FORMAT_UI);
+            this.fromDate = moment(value.startDate).format(GIDDH_DATE_FORMAT);
+            this.toDate = moment(value.endDate).format(GIDDH_DATE_FORMAT);
+            this.mfStockSearchRequest.from = this.fromDate;
+			this.mfStockSearchRequest.to = this.toDate;
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/manufacturing/report/mf.report.component.ts:348]: Already formatted but appearing unformatted on GIthub diff view

File: apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html [hunk 3]
```diff
@@ -593,12 +591,8 @@ <h3 class="modal-title bg" id="modal-title">Advance Search</h3>
                     <div class="row">
                         <div class="col-sm-4">Date Range</div>
                         <div class="col-sm-8">
-                            <!-- <div class="input-group">
-                                <input type="text" name="daterangeInput1" daterangepicker [options]="datePickerOptions" class="form-control" (applyDaterangepicker)="selectedDate($event, 'advance')" />
-                                <div class="input-group-addon"><span class="icon-calender bg-white"></span></div>
-                            </div> -->
                             <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
-                                <i><img src="assets/images/custom-calender.svg"></i>
+                                <i></i>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html:595]: what's this????

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html (hunk 3 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html]: resolved

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**